# Copyright (c) 2006, Oracle and/or its affiliates. All rights reserved.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
#

Part of fix for bug 5077065: Events appear even if XevieSendEvent is not called

--- a/XKBsrv.h	Thu Sep  1 12:56:14 2005
+++ b/XKBsrv.h	Fri Apr 21 19:31:12 2006
@@ -293,6 +293,7 @@
 
 extern CARD32	xkbDebugFlags;
 extern CARD32	xkbDebugCtrls;
+extern int	xkbDevicePrivateIndex;
 
 #define	_XkbAlloc(s)		xalloc((s))
 #define	_XkbCalloc(n,s)		Xcalloc((n)*(s))
@@ -768,6 +769,13 @@
     int 			/* count */
 );
 
+extern void xkbUnwrapProc(
+    DeviceIntPtr		/* device */,
+    DeviceHandleProc 		/* unwrap proc */,
+    pointer 			/* data */
+);
+
+
 extern Bool XkbEnableDisableControls(
     XkbSrvInfoPtr	/* xkbi */,
     unsigned long	/* change */,
